lldb/source/Core/ValueObject.cpp

```diff
@@ -2763,10 +2763,10 @@ ValueObjectSP ValueObject::Cast(const CompilerType &compiler_type) {
   ExecutionContextScope *exe_scope
       = ExecutionContext(GetExecutionContextRef())
           .GetBestExecutionContextScope();
-  if (compiler_type.GetByteSize(exe_scope)
-      <= GetCompilerType().GetByteSize(exe_scope) 
-      || m_value.GetValueType() == Value::ValueType::LoadAddress)
-        return DoCast(compiler_type);
+  if (compiler_type.GetByteSize(exe_scope) <=
```

Comment:
This is just a reformatting, right?  We try to only clang-format the code the PR actually changes, (using git-clang-format) as it makes reviews easier.  Also clang-format changes its mind about how to format over time, and so doing other code introduces spurious diffs that make archeology harder.

Reply:
Hmm, yeah I ran `git-clang-format` against this. 
Specifically, `git fetch upstream; git-clang-format upstream/main` and it produced this. I'll manually revert this one